pkg/operator/ceph/csi/controller.go

```diff
@@ -331,13 +329,10 @@ func (r *ReconcileCSI) reconcile(request reconcile.Request) (reconcile.Result, e
 				return opcontroller.ImmediateRetryResult, errors.Wrap(err, "failed to reconcile csi-op config CR")
 			}
 			return reconcileResult, nil
-		} else {
-			r.deleteCSIOperatorResources(clusterNamespace, false)
 		}
 	}
 
 	if !EnableCSIOperator() {
-		r.deleteCSIOperatorResources(clusterNamespace, true)
```

Comment:
if csi operator is enable by rook,
And then it is disabled,

Who will delete these resources?

Reply:
It will be user responsibility to delete the resources